src/services/PugQueue.js

```diff
@@ -25,35 +26,34 @@ export class PugQueue {
     this.queue.push(member)
     member.addRole(process.env.PUGS_ROLE)
 
-    return `${String.fromCodePoint(0x2705)} ${name} added to queue. ${this.getQueueState()}`
+    return `${String.fromCodePoint(0x2705)} ${name} ${roleEmoji} added to queue. ${this.getQueueState()}`
```

Comment:
I don't think we need to put the role emoji in this message, would rather keep the 'noisiness' of the bot's messages to a minimum. The roles are only really relevant in the announcements message when people are trying to make teams.